server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -2585,10 +2585,7 @@ public NetworkResponse createNetworkResponse(ResponseView view, Network network)
                 Domain domain = ApiDBUtils.findDomainById(domainNetworkDetails.first());
                 if (domain != null) {
                     response.setDomainId(domain.getUuid());
-
-                    StringBuilder domainPath = new StringBuilder("ROOT");
-                    (domainPath.append(domain.getPath())).deleteCharAt(domainPath.length() - 1);
-                    response.setDomainPath(domainPath.toString());
+                    response.setDomainPath(getDomainPath(domain.getPath()));
```

Comment:
Didn't test but will this cause a change in response with `ROOT` not getting prepended

Reply:
is there any functional reason to prepend "ROOT"? It is already equivilent with "/", functionally and the current code returns an empty string.

Reply:
@DaanHoogland cc @isupersky I'm okay either. My only concern was that users/API consumers would start seeing a difference. Will that be an issue?

Reply:
I think "ROOT" is a representational depiction of "/", and should be only shown in the UI. It makes no sense in the API and leads to less convenient processing. I would not want it in an API. Right now the response parameter `domainpath` is empty. I tested the difference:
```
(localcloud) 🐱 > list networks filter=domainpath
```
before:
```
{
  "count": 1,
  "network": [
    {}
  ]
}
```
after:
```
{
  "count": 1,
  "network": [
    {
      "domainpath": "/"
    }
  ]
}
```